src/pages/sources/SourcesListView.tsx

```diff
@@ -261,6 +261,15 @@ const SourcesListView: React.FunctionComponent = () => {
       .finally(() => setPendingDeleteSource(undefined));
   };
 
+  const hasSelectedSources = () => {
+    return Object.values(selectedItems).filter(val => val !== null).length > 0;
```

Comment:
trying to figure out what all of this is. shouldn't `selectedItems` already be an array and you could just do `selectedItems?.length`?

Reply:
well, I didnt know better. I will push mods and ask for re-review